vllm/v1/executor/ray_distributed_executor.py

```diff
@@ -37,6 +53,80 @@ def max_concurrent_batches(self) -> int:
             return 2
         return self.parallel_config.pipeline_parallel_size
 
+    def _aggregate_workers_output(
```

Comment:
Can we move these to utility functions (shared with multiproc executor), or maybe a mixin superclass also containing the `_send_remaining_count` and `_recv_remaining_count` fields?